src/views/RecIM/components/List/Listing/index.jsx

```diff
@@ -412,6 +455,32 @@ const ParticipantListing = ({
             )}
           </Menu>
         )}
+        {isAdminPage && participant.IsCustom && (
+          <Menu
+            open={moreOptionsCustomOpen}
+            onClose={handleClickOff}
+            anchorEl={anchorCustomEl}
```

Comment:
can we rename
```
moreOptionsCustomOpen
anchorCustomEl
```
to something else? Those variable names are not very descriptive. moreParticipantOptions? participantAnchorEl?